compiler/aarch64/codegen/ARM64SystemLinkage.cpp

```diff
@@ -555,9 +555,9 @@ TR::ARM64SystemLinkage::createPrologue(TR::Instruction *cursor, List<TR::Paramet
 void
 TR::ARM64SystemLinkage::createEpilogue(TR::Instruction *cursor)
    {
-   TR::CodeGenerator *codeGen = cg();
+   TR::CodeGenerator *cg = cg();
```

Comment:
The compiler can't determine if `cg` refers to the function or the auto, so you need to introduce a `this->cg()` to remove the ambiguity.  This happens in a number of places throughout this PR.